### DISCUSSION TITLE
Import error in third party dependency

### DISCUSSION THREAD

**Discussion by KeosWang** [NatLabRockies/floris#884] [Q&A]
Hello Author

It is a great honour to be able to use the package you have developed. Recently I have encountered some problems while using it. I downloaded a wakenet programme from github, and when I run the programme, it always shows that it can't call the static_class programme in floris, it shows that it can't find the file. The author asked to use floris 2.4. and I didn't find the corresponding file in there. 

So I have no choice but to ask for help here. Is there such a file in floris, and how can I debug it to solve this bug?

Attached is a report of the error.
```
Traceback (most recent call last).
  File "c:\Users\lll\Desktop\Code\python_code\wakenet-master\Code\example_main.py", line 1, in <module>
    from neuralWake import *
  File "c:\Users\lll\Desktop\Code\python_code\wakenet-master\Code\neuralWake.py", line 2, in <module>
    from initialisations import *
  File "c:\Users\lll\Desktop\Code\python_code\wakenet-master\Code\initialisations.py", line 5, in <module>
    from floris.tools import static_class as sc
ImportError: cannot import name 'static_class' from 'floris.tools' (D:\anaconda\lib\site-packages\floris\tools\__init__.py)
```

Hope to hear from you soon!

Best Regards，
Keos
![捕获](https://github.com/NREL/floris/assets/149571247/57f383d1-d9af-4680-9694-54264842dc96)

**Comment by paulf81** [comment]
Hi Keos, I'm sorry I'm unfamiliar with the program wakenet.  The line of code causing the issue appears to be this one:

```
from floris.tools import static_class as sc
```

I can't remember us having such an import available, perhaps wakenet is using a modified version of floris?  @rafmudaf, do you remember an import like this in v2?

**Reply by rafmudaf** [reply]
No, I don't think there was ever a `static_class` in FLORIS - at least not that I recall. They're only setting two attributes on that object and it's never read from (see [here](https://github.com/soanagno/wakenet/blob/master/Code/initialisations.py#L112-L115)), so I suggest to simply comment those lines. Specifically, try commenting these:
- https://github.com/soanagno/wakenet/blob/master/Code/initialisations.py#L5
- https://github.com/soanagno/wakenet/blob/master/Code/initialisations.py#L113
- https://github.com/soanagno/wakenet/blob/master/Code/initialisations.py#L114
